docs/tables/aws_guardduty_finding/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Tailpipe Table: aws_guardduty_finding - Query AWS GuardDuty Findings"
+description: "AWS GuardDuty findings provide alerts and intelligence about potential security threats and suspicious activities detected in your AWS environment."
+---
+
+# Table: aws_guardduty_finding - Query AWS GuardDuty Findings
+
+The `aws_guardduty_finding` table allows you to query data from AWS GuardDuty findings. This table provides detailed information about potential security threats detected within your AWS environment, including the affected resources, severity levels, finding types, and contextual details to help you investigate and respond to security issues.
+
+## Configure
+
+Create a [partition](https://tailpipe.io/docs/manage/partition) for `aws_guardduty_finding` ([examples](https://hub.tailpipe.io/plugins/turbot/aws/tables/aws_guardduty_finding#example-configurations)):
+
+```sh
+vi ~/.tailpipe/config/aws.tpc
+```
+
+```hcl
+connection "aws" "security_account" {
+  profile = "my-security-account"
+}
+
+partition "aws_guardduty_finding" "my_findings" {
+  source "aws_s3_bucket" {
+    connection = connection.aws.security_account
+    bucket     = "aws-guardduty-findings-bucket"
+  }
+}
+```
+
+## Collect
+
+[Collect](https://tailpipe.io/docs/manage/collection) findings for all `aws_guardduty_finding` partitions:
+
+```sh
+tailpipe collect aws_guardduty_finding
+```
+
+Or for a single partition:
+
+```sh
+tailpipe collect aws_guardduty_finding.my_findings
+```
+
+## Query
+
+**[Explore example queries for this table →](https://hub.tailpipe.io/plugins/turbot/aws/queries/aws_guardduty_finding)**
+
+### High Severity Findings
+
+List all high severity security findings to prioritize your security response.
+
+```sql
+select
+  tp_timestamp,
+  tp_index as account_id,
+  region,
+  title,
+  type,
+  severity,
+  description
+from
+  aws_guardduty_finding
+where
+  severity >= 7.0
+order by
+  severity desc,
+  tp_timestamp desc;
+```
+
+### Findings by Type
+
+Group findings by type to understand the most common security threats in your environment.
+
+```sql
+select
+  type,
+  count(*) as finding_count,
+  min(created_at) as first_seen,
+  max(created_at) as last_seen,
+  round(avg(severity), 2) as avg_severity
+from
+  aws_guardduty_finding
+group by
+  type
+order by
+  finding_count desc;
+```
+
+### Recent Findings with Resource Details
+
+Examine recent security findings along with details of the affected resources.
+
+```sql
+select
```

Comment:
Please see comments in `queries.md` and apply the same changes here (column ordering, using dot notation, etc.)